tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2017",
```

Comment:
Maybe we should change the ecmaScript version, but I feel like this should be in a different PR.